tests/puzzle/test_solver.py

```diff
@@ -1858,8 +1889,11 @@ def test_solver_does_not_trigger_conflict_for_python_constraint_if_python_requir
     check_solver_result(transaction, [{"job": "install", "package": package_a}])
 
 
-def test_solver_does_not_trigger_conflict_for_python_constraint_if_python_requirement_is_compatible_multiple(  # noqa: E501
-    solver: Solver, repo: Repository, package: Package
+def test_solver_does_not_trigger_conflict_for_python_constraint_if_python_requirement_is_compatible_multiple(
+    # noqa: E501
+    solver: Solver,
+    repo: Repository,
+    package: Package,
```

Comment:
```suggestion
    package: Package
```

Not sure why this trailing comma snuck in, but let's drop it.